cclib/parser/turbomoleparser.py

```diff
@@ -108,8 +108,8 @@ def extract(self, inputfile, line):
         if line[3:11] == "nbf(AO)=":
             nmo = int(line.split('=')[1])
             self.set_attribute('nbasis', nmo)
-            self.set_attribute('nmo', nmo)
-
+            #self.set_attribute('nmo', nmo)

```

Comment:
Should this be deleted?

Reply:
Yes it is no longer required; there is an alternative method now for setting nmo